hbase-shell/src/test/ruby/hbase/admin_test.rb

```diff
@@ -528,274 +528,5 @@ def teardown
       end
     end
   end
-
-  # Simple administration methods tests
-  class AdminRegionTest < Test::Unit::TestCase
```

Comment:
Why we remove this stuff?

Reply:
moved to admin3_test.rb. Split it and make every test run less than 10 min.